addons/core/translations/states/en-us.yaml

```diff
@@ -18,3 +18,6 @@ loading:
   title: Loading all items...
   description: All of your {resource} are still loading. Please wait while we finish loading everything on screen.
   generic-description: All of your resources are still loading. Please wait while we finish loading everything on screen.
+incomplete-loading:
+  limit: We've reached the result limit. Please narrow down your search or filters if the result you're looking for is not shown.
```

Comment:
None blocking: it is "reached the results limit" or "reached the result limit"? Asking because in the `refreshing` string we say "results"

Reply:
Hmm I was referring to "results" in the other statement in the plural sense but in this case I did mean the "result limit" in the singular sense (i.e. the result limit is 250, a singular number)